WordPress/Classes/Services/BlogService.m

```diff
@@ -510,6 +510,11 @@ - (NSArray *)blogsWithNoAccount
     return [self blogsWithPredicate:nil];
 }
 
+- (NSArray<Blog *> *)blogsForWPComAccounts
+{
+    return [self blogsWithPredicate:[NSPredicate predicateWithFormat:@"account != NULL"]];
```

Comment:
The web reblog doesn't show hidden sites. I don't think the app should either (pinging @frosty for confirmation). I don't see an existing method for doing this (blog list has it's own logic), so I think we could update this to be  `visibleBlogsForWPComAccounts`, and include the `predicateForVisibleBlogs`.

Reply:
Yes, even more so, the blog selector already filters the hidden sites.

Reply:
I agree, we should match what the web is doing 👍